### DISCUSSION TITLE
ansible-pull error in frontend

### DISCUSSION THREAD

**Discussion by krishnaja-b** [learndevopsonline/learndevopsonline#1106] [Q&A]
Hello Sir,

When I am pulling ansible in the frontend, after installation of node exporter, I am getting the below error.
![2](https://github.com/learndevopsonline/learndevopsonline/assets/125537894/7deeb6c3-04b0-4240-97d9-4f6b27721b5e).

But I created vars directory.
![1](https://github.com/learndevopsonline/learndevopsonline/assets/125537894/57c6ae8d-8cd1-4650-a654-9648e17c5605)

Regards,
Krishnaja.

**Comment by Akhileshreddy9181** [comment]
"-" is not needed for declaring the variables in Ansible.
Please recheck the syntax.

**Reply by krishnaja-b** [reply]
Hi
I solve the issue. I go through the old discussions, and based on that I change the frontend instance type to t3.small.
Then I can able to see the frontend on Prometheus.
